src/ecs-patterns/application-load-balanced-codedeployed-fargate-service.ts

```diff
@@ -105,6 +106,13 @@ export interface ApplicationLoadBalancedCodeDeployedFargateServiceProps extends
    * @default - listenerPort + 1
    */
   readonly testPort?: number;
+
+  /**
+   * Optional lifecycle hooks
+   *
+   * @default - no lifecycle hooks
+   */
+  readonly hooks?: AppSpecHooks;
```

Comment:
This should be passed to the `EcsDeployment()`

Reply:
Good spot. Done